code/modules/shuttle/shuttle.dm

```diff
@@ -320,17 +361,60 @@
 	if(smooth)
 		smooth_icon(src)
 
+/obj/docking_port/mobile/proc/jumpToNullSpace()
+	// Destroys the docking port and the shuttle contents.
+	// Not in a fancy way, it just ceases.
+
+	JUSTDOIT = TRUE //GOD WHY WHY GOD
```

Comment:
This means when you roll over the tile that contains this docking port while deleting stuff, you'll delete it, and stop this proc cold. Put this at the end, before the `qdel(src)` instead.